bin/ruby-build

```diff
@@ -1256,6 +1256,29 @@ build_package_ldflags_dirs() {
   done
 }
 
+build_package_enable_yjit() {
+  local rustc_installed rustc_ver
+
+  rustc_installed=""
+  if which rustc; then
+    rustc_ver=`rustc --version | cut -c7-10`
+    # Some kind of built-in bash comparison for dotted version strings would be awesome.
+    if [[ " ${rustc_ver} " == "1."[6789]* ]]; then
```

Comment:
```suggestion
    if [[ "${rustc_ver}" == "1."[6789]* ]]; then
```
Otherwise the extra spaces probably prevent matching, no?

Reply:
Ah, I had pushed a less-than-latest version without realising. D'oh! Fixed now.